lib/stat_tracker.rb

```diff
@@ -18,4 +18,32 @@ def self.from_csv(locations = {games: './data/games_sample.csv', teams: './data/
     GameTeam.from_csv(locations[:game_teams])
     self.new(Team.all_teams, Game.all_games, GameTeam.all_game_teams)
   end
+
+# ~~~ HELPER METHODS~~~
+  def total_games
+    @games.count
+  end
+
+# ~~~ GAME METHODS~~~
+  def percentage_away_wins
+    wins = @games.find_all { |game| game.away_goals > game.home_goals}
+    (wins.count / total_games.to_f * 100).round(2)
+  end
+
+  def percentage_ties
+    ties = @games.find_all { |game| game.away_goals == game.home_goals}
+    (ties.count / total_games.to_f * 100).round(2)
+  end
+
+  def percentage_home_wins
+    wins = @games.find_all { |game| game.away_goals < game.home_goals}
+    (wins.count / total_games.to_f * 100).round(2)
+  end
```

Comment:
I wonder if a helper method could be used to calculate percentage? It looks like we will need something like that at least one other time.